src/app/map/map.component.svg

```diff
@@ -0,0 +1,5119 @@
+	<line class="st0" x1="707.97" y1="518" x2="736.4" y2="518"/>
+	<line class="st0" x1="736.4" y1="530.29" x2="736.4" y2="518"/>
+	<line class="st0" x1="742.72" y1="532.92" x2="736.4" y2="530.29"/>
+	<line class="st0" x1="760.62" y1="539.76" x2="765.36" y2="542.13"/>
+	<line class="st0" x1="773.79" y1="519.23" x2="765.36" y2="542.13"/>
+	<line class="st0" x1="966.82" y1="518.61" x2="773.79" y2="519.23"/>
+	<line class="st0" x1="974.41" y1="541.87" x2="966.82" y2="518.61"/>
+	<line class="st0" x1="1027.06" y1="518.92" x2="994.42" y2="532.66"/>
+	<line class="st0" x1="1027.06" y1="518.92" x2="1025.59" y2="729.46"/>
+	<line class="st0" x1="1238.3" y1="728.87" x2="1025.59" y2="729.46"/>
+	<line class="st0" x1="390.86" y1="183" x2="390.86" y2="29"/>
+	<line class="st0" x1="813.86" y1="29" x2="390.86" y2="29"/>
+	<line class="st0" x1="424.86" y1="181.94" x2="603.16" y2="181.94"/>
+	<line class="st0" x1="603.22" y1="193.4" x2="602.35" y2="29"/>
+	<line class="st0" x1="624.77" y1="192.5" x2="789.85" y2="191.97"/>
+	<line class="st0" x1="813.86" y1="29" x2="813.86" y2="202.24"/>
+	<line class="st0" x1="844.35" y1="202.24" x2="992.31" y2="202.24"/>
+	<line class="st0" x1="1202.86" y1="29" x2="813.86" y2="29"/>
+	<line class="st0" x1="1023.64" y1="29" x2="1024.17" y2="202.24"/>
+	<line class="st0" x1="1202.86" y1="29" x2="1202.86" y2="225"/>
+	<line class="st0" x1="1202.86" y1="225" x2="1077.86" y2="225"/>
+	<line class="st0" x1="1077.86" y1="194.61" x2="1077.86" y2="29"/>
+	<line class="st0" x1="1098.86" y1="302" x2="1098.86" y2="268.06"/>
+	<line class="st0" x1="1057.86" y1="268.06" x2="1098.86" y2="268.06"/>
+	<line class="st0" x1="1057.86" y1="290.44" x2="1057.86" y2="302"/>
+	<line class="st0" x1="1057.86" y1="302" x2="1098.86" y2="302"/>
+	<line class="st0" x1="1098.86" y1="434" x2="1098.86" y2="302"/>
+	<line class="st0" x1="1057.86" y1="325.19" x2="1057.86" y2="342.57"/>
+	<line class="st0" x1="1057.86" y1="360.74" x2="1057.86" y2="378"/>
+	<line class="st0" x1="1057.86" y1="396.45" x2="1057.86" y2="434"/>
+	<line class="st0" x1="1098.86" y1="434" x2="1057.86" y2="434"/>
+	<line class="st0" x1="1098.86" y1="337.83" x2="1057.86" y2="337.57"/>
+	<line class="st0" x1="1098.88" y1="372.85" x2="1057.86" y2="372.85"/>
+	<line class="st0" x1="1057.86" y1="417.08" x2="1039.96" y2="417.08"/>
+	<line class="st0" x1="1021.53" y1="417.08" x2="1014.86" y2="417.08"/>
+	<line class="st0" x1="1014.86" y1="290.44" x2="1014.86" y2="476"/>
+	<line class="st0" x1="818.86" y1="280" x2="818.86" y2="476"/>
+	<line class="st0" x1="835.4" y1="280" x2="818.86" y2="280"/>
+	<line class="st0" x1="865.94" y1="280" x2="1014.86" y2="280"/>
+	<line class="st0" x1="1014.86" y1="290.44" x2="1014.86" y2="280"/>
+	<line class="st0" x1="710.47" y1="728" x2="711.51" y2="518"/>
+	<line class="st0" x1="1202.86" y1="29" x2="1234.32" y2="29"/>
+	<line class="st0" x1="1238.3" y1="728.87" x2="1234.32" y2="29"/>
+</g>
+<g id="Layer_2" class="st9">
```

Comment:
I'm not sure if everything within this <g></g> needs to be here. When I comment it out it seems to still work.